### DISCUSSION TITLE
posts/2024-01-09-insects/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#2] [Announcements]
# posts/2024-01-09-insects/

I learned several fascinating and surprising facts about insects in my microbiology courses.

https://rachel.fast.ai/posts/2024-01-09-insects/

**Comment by jesse-sealand** [comment]
Insect cells make for cheaper and faster expression systems than mammalian cells. That's really interesting to me and especially that their proteins are more likely to fold correctly. This is kind of amazing!

**Reply by racheltho** [reply]
Yes, I was amazed to learn about insect/baculovirus expression systems too!
